simple_history/models.py

```diff
@@ -111,7 +111,10 @@ def create_history_model(self, model, inherited):
         """
         Creates a historical model to associate with the model provided.
         """
-        attrs = {'__module__': self.module}
+        attrs = {
+            '__module__': self.module,
+            'excluded_fields': self.excluded_fields
```

Comment:
This prevents someone from using `excluded_fields` as a field_name, no?

Reply:
Or it will get overwritten by that field name, but either way it's not safe.

Reply:
Could we add it to the Meta class of the historical model instead?

Reply:
If that moved to meta class that will lead to exception by Django. If that fixed some how that will not be detected by Django migrator and lead to excluded fields being ignored within Django Migrations.

I'm adding a `_` prefix to `excluded_fields` thus user can have a field with that name.